connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/RestClient.java

```diff
@@ -190,15 +190,15 @@ private <T> HttpResponse<T> httpRequest(HttpClient client, String url, String me
                         "Unexpected status code when handling forwarded request: " + responseCode);
             }
         } catch (IOException | InterruptedException | TimeoutException | ExecutionException e) {
-            log.error("IO error forwarding REST request: ", e);
+            log.error("IO error forwarding REST request to URL: " + url, e);
```

Comment:
Instead of string concatenation, can we use standard Log4J formatting syntax?

```suggestion
            log.error("IO error forwarding REST request to {}", url, e);
```

Reply:
@C0urante Thanks for the review; Applied.